src/main/java/io/github/syst3ms/skriptparser/lang/Expression.java

```diff
@@ -32,8 +33,8 @@
     /*
      * This is staying until we figure out a better way to implement this
      */
-    default T[] getArray(TriggerContext e) {
-        return getValues(e);
+    default Optional<T[]> getArray(TriggerContext e) {
+        return Optional.ofNullable(getValues(e));
```

Comment:
`getValues()` is never null, so changing this method to an Optional shouldn't be necessary.